src/rlookup.c

```diff
@@ -884,6 +1042,7 @@ int RLookup_LoadRuleFields(RedisModuleCtx *ctx, RLookup *it, RLookupRow *dst, In
                             .noSortables = 1,
                             .mode = RLOOKUP_LOAD_KEYLIST };
   int rv = loadIndividualKeys(it, dst, &opt);
+  QueryError_ClearError(&status);
```

Comment:
Should this be in a separate PR so it could be cherry-picked?

Reply:
It can be, although it's not a real leak as nobody sets an error message in `loadIndividualKeys` today, it's just for good practice and future proof. Let me know if you'd like me to move it anyway.

Reply:
Then we can do it in this PR.